guides/tutorials/codelabs/index.md

```diff
@@ -9,6 +9,7 @@ Run through these codelabs to get hands-on experience with Spinnaker in a guided
 
 ## Table of Contents
 
+* [Continuous Delivery to Kubernetse Using Spinnaker (Google I/O 2018 Spinnaker Codelab)](https://codelabs.developers.google.com/codelabs/cloud-spinnaker-kubernetes-cd/index.html?index&authuser=0#0) - deploy an application from Kubernetes manifest files
```

Comment:
[minor] Curious about the `?index&authuser=0#0` part of the URL - is that needed?

Also, from a brief glance, Google I/O 2018 isn't obviously mentioned - perhaps call it "Kubernetes manifest sample pipelines" or so?

/cc @lwander

Reply:
Yep, cleaned up the URL, thanks. Also the link description does have Google I/O 2018 in it. Is that what you mean?

Reply:
> Also the link description does have Google I/O 2018 in it. Is that what you mean?

I meant the "Google I/O 2018 Spinnaker Codelab" description, yes.

Reply:
Done.